CODEOWNERS

```diff
@@ -22,8 +22,9 @@ topics/ecology/                    @galaxyproject/training-ecology
 topics/epigenetics/                @galaxyproject/training-epigenetics
 topics/galaxy-interface/           @galaxyproject/training-galaxy-ui @galaxyproject/training-galaxy-data-manipulation
 topics/genome-annotation/          @galaxyproject/training-genome-annotation
-topics/teaching/                @galaxyproject/training-instructors
+topics/teaching/                   @galaxyproject/training-instructors
 topics/introduction/               @galaxyproject/training-introduction
+topics/materials-science/          @galaxyproject/training-materials-science
```

Comment:
Since `materials-science` is a new topic (I strongly assume) there isn't a team for it under `galaxyproject`. I suppose we would ask for this to be created as part of the upstream PR?